### PR TITLE
MAINT: unit test standards

### PR DESCRIPTION
**Pull request by jklenzing**
Updates the unit test area, including:
- use of `num_samples` keyword
- docstring standards
- return statements

Also updates the workflow to use numpy 1.18 as the minimum test version

### CODE REVIEW

File: pysatIncubator/tests/test_instruments.py [hunk 1]
```diff
@@ -52,15 +70,12 @@ def setup_class(self):
...
         pysat.params.data['data_dirs'] = self.saved_path
         self.tempdir.cleanup()
-        del self.inst_loc, self.saved_path, self.tempdir
-
-    def setup_method(self):
-        """Runs before every method to create a clean testing setup."""
-
-    def teardown_method(self):
-        """Runs after every method to clean up previous testing."""
+        del self.inst_loc, self.saved_path, self.tempdir, self.stime
```
Comment (jklenzing) [inline pysatIncubator/tests/test_instruments.py]:
```suggestion
        del self.inst_loc, self.saved_path, self.tempdir
```

File: pysatIncubator/tests/test_methods_empirical.py [hunk 2]
```diff
@@ -70,5 +87,6 @@ def test_add_msis(self):
             assert not np.isnan(self.testInst[target]).any()
             # Check if metadata is added
             assert target in self.testInst.meta.data.index
+        return
 
     # TODO: Add hwm tests once routine is generalized
```
Comment (aburrell) [inline pysatIncubator/tests/test_methods_empirical.py]: Is this captured in an issue?

Comment (jklenzing) [inline pysatIncubator/tests/test_methods_empirical.py]:
```suggestion
```

Comment (jklenzing) [reply pysatIncubator/tests/test_methods_empirical.py]: deleting since I don't know whether we will maintain pyglow support.

**Review by aburrell — APPROVED** [review]

**Pull request merged**